fatiando/mesher/geometry.py

```diff
@@ -92,6 +94,57 @@ def x(self):
     def y(self):
         return self.vertices[:, 1]
 
+    @property
+    def orientation(self):
+        area = self.get_area(absolute=False)
+        if area < 0:
+            return "CW"
+        else:
+            return "CCW"
+
+    def set_clockwise(self):
+        self.set_orientation("CW")
+
+    def set_orientation(self, orientation):
+        if orientation not in ["CW", "CWW"]:
+            raise ValueError("Orientation must be 'CW' or 'CCW'")
+        current = self.orientation
+        if orientation != current:
+            self._vertices = self._vertices[::-1]
+
+    def get_area(self, absolute=True):
```

Comment:
This should probably be a private method because the `absolute` parameter doesn't make a lot of sense from a user perspective (but we need it for orientation). I would suggest turning this into `_compute_area` without the absolute value. Then we can use this to calculate the orientation. And having an `area` property that calls `_compute_area` and applies the absolute value always.